stable/fairwinds-insights/README.md

```diff
@@ -5,7 +5,7 @@
 > The self-hosted version of Fairwinds Insights is currently in alpha.
 > The documentation is incomplete, and it is subject to breaking changes.
 
-See [insights.docs.fairwinds.com](https://insights.docs.fairwinds.com/self-hosted) for complete documentation.
+See [insights.docs.fairwinds.com](https://insights.docs.fairwinds.com/self-hosted/installation/) for complete documentation.
```

Comment:
Oh you need to update the gotmpl, then you could rerun helm-docs to update the readme itself.